docs/protocol-spec.md

```diff
@@ -2207,14 +2207,14 @@ as depicted in Sections 4.7.1 and 4.7.2.
  - Replay attack resistance mechanism is not available
  - Invocation access control can be enforced at the application layer:
    it is up to the application to leverage the infrastructure's tools properly
-   for security to be guaranteed. This means, that if the application ignores
-   to *bind* the transaction binding offered by our fabric, secure transaction
-   processing  may be at risk.
+   for security to be guaranteed. This means, that if the application fails to
+   *bind* the transaction binding offered by the fabric, secure transaction
+   processing may be at risk.
 
 #### 4.7.1 Simplified client
 
-Client side enrollment and transaction creation is performed entirely by a non-validating peer who plays the role of an online wallet.
-In particular, the end-user leverages his registration credentials <username, password> to open an account to a non-validating peer
+Client side enrollment and transaction creation is performed entirely by a non-validating peer that plays the role of an online wallet.
```

Comment:
**Client-side** enrollment and transaction creation **are** performed entirely by a non-validating peer **which** plays the role of an online wallet.